packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/setConfig.ts

```diff
@@ -24,7 +26,82 @@ type ContractInput = [
   offchain_config: string[],
 ]
 
-const makeContractInput = async (input: SetConfigInput): Promise<ContractInput> => {
+const makeUserInput = async (flags, args, env): Promise<SetConfigInput> => {
+  if (flags.input) return flags.input as SetConfigInput
+  if (flags.rdd) {
+    const rdd = getRDD(flags.rdd)
+    const contractAddr = args[0]
+    const contract = rdd[CONTRACT_TYPES.AGGREGATOR][contractAddr]
+    const config = contract.config
+    const operators = contract.oracles.map((oracle) => rdd.operators[oracle.operator])
+    const publicKeys = operators.map((o) =>
+      o.ocr2OffchainPublicKey[0].replace('ocr2off_starknet_', ''),
```

Comment:
I'd propose all the replacements (i.e. content transformations) to happen in `makeContractInput` for consistency.

Reply:
fair, looks like the replace was doubled up anyway in `makeContractInput` so removed this one 👍